### PR TITLE
Simplify docker images

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

**Comment by electrum** [comment]
Do you want someone to review this? Otherwise I’m fine to just merge this.

What do we need to update the product tests to use it?

**Comment by electrum** [comment]
BTW, why the name “prestosql”? Using “presto” or “prestodb” is preferable since that’s what we use elsewhere. We use “prestodb” when “presto” isn’t available (such as the domain or GitHub organization).

Thanks for doing this!

**Comment by arhimondr** [comment]
Both presto and prestodb were already taken =\

**Comment by arhimondr** [comment]
> What do we need to update the product tests to use it?

I'm still working on documentation and some nits. Once i'm done i will open a PR to Presto

File: prestosql/cdh5-base/Dockerfile [hunk 1]
```diff
@@ -48,7 +48,6 @@ RUN wget -nv http://archive.cloudera.com/cdh5/one-click-install/redhat/6/x86_64/
 COPY files/startup.sh /root/startup.sh
 COPY files/supervisord.conf /etc/supervisord.conf
 COPY files/supervisord.d/bootstrap.conf /etc/supervisord.d/bootstrap.conf
-COPY files/change_timezone.sh /root/change_timezone.sh
```
Comment (losipiuk) [inline prestosql/cdh5-base/Dockerfile]: This is needed by WIP work on fixing timestamp semantics by @fiedukow. So tests do not use UTC which tends to hide bugs.
I have high hopes that work will be picked up by someone so please leave this.

**Comment by arhimondr** [comment]
I set the timezone to Asia/Katmandu in all the images  by default. Does it
sound good? Or do we need to change timezone somewhere conditionally on the
go?

**Comment by losipiuk** [comment]
> I set the timezone to Asia/Katmandu in all the images  by default. Does it
sound good? Or do we need to change timezone somewhere conditionally on the
go?

Oh - then this is probably fine. @fiedukow ?

**Comment by findepi** [comment]
> Both presto and prestodb were already taken =\

You mean @ https://hub.docker.com/ ? https://hub.docker.com/u/prestodb/ looks like Presto's, maybe it's something @electrum has access to?

**Comment by findepi** [comment]
> Backup hive kerberized configuration

why are you renaming cdh* files to hdp* files? 

> Remove CDH based images

is cdh no longer "supported"?

> https://hub.docker.com/u/prestosql/

(if this is needed because https://hub.docker.com/u/prestodb/ is not accessible...)
Is it a user, not an organization? Who has access?

Sorry, I didn't do proper review. I know too little about the various configuration issues this PR touches.

**Comment by kokosing** [comment]
AFAIR I reserved prestodb. Send me your login and I will try to give you ab access.

**Comment by fiedukow** [comment]
@losipiuk @arhimondr I thought we want to keep those docker images universal (not bound to usage in product-tests), so I wanted to avoid changing images default (as UTC seems reasonable and Asia/Kathmandu seems a bit random). But for sure it should work in product-tests if you'll just change the defaults.

**Comment by arhimondr** [comment]
@findepi 

> why are you renaming cdh* files to hdp* files?

This is the first step of migrating kerberazied image based on CDH to HDP. There are two steps. 

- Move everything
- Fix to make it work

> is cdh no longer "supported"?

We may support this in a future. For now since we don't know what are the resources will be allocated to that it is better to keep it small, just to make the maintenance process as simple as possible. So it can live.

@fiedukow 

Let's keep it simple for now. We may change it in the future.

**Pull request merged**